src/pages/[userId]/prospects.js

```diff
@@ -1,54 +1,148 @@
 import { useRouter } from 'next/router';
-import Grid from '../../components/utils/Grid';
+import styled from '@emotion/styled';
+import {
+  pipe, join, length, path, prop,
+} from 'ramda';
+import Grid from '@material-ui/core/Grid';
```

Comment:
You can also delete our old custom `Grid` component along with the `utils` directory.